lib/modules/versioning/distro.spec.ts

```diff
@@ -1,8 +1,18 @@
+import { DateTime, Settings } from 'luxon';
 import { DistroInfo } from './distro';
 
 describe('modules/versioning/distro', () => {
   const di = new DistroInfo('data/ubuntu-distro-info.json');
 
+  beforeAll(() => {
+    const dt = DateTime.fromISO('2022-03-20');
```

Comment:
```suggestion
    const dt = DateTime.fromISO('2021-03-20');
```

we should have more unreleased versions for better testing, because the `getNLatest` seems to misbahave then 😉